contrib/kafka/pkg/adapter/adapter.go

```diff
@@ -77,8 +77,9 @@ func (a *Adapter) ConsumeClaim(sess sarama.ConsumerGroupSession, claim sarama.Co
 	logger := logging.FromContext(context.TODO())
 
 	for msg := range claim.Messages() {
-		logger.Info("Received: ", zap.Any("value", string(msg.Value)))
-
+		logger.Info("Received: ", zap.String("topic:", string(msg.Topic)),
```

Comment:
I think this should be `Debug` logging. Logging every message at `Info` can cause performance issues at high throughput.

Reply:
Sure, addressed in upcoming commit